Scripts/Runtime/Entities/TaskDriver/TaskSet/Job/JobConfig/AbstractJobConfig.cs

```diff
@@ -284,6 +291,13 @@ public void Harden()
 
             foreach (AbstractAccessWrapper wrapper in m_AccessWrappers.Values)
             {
+                //Only allow one wrapper per type for DataStream Pending Access since they will all try to acquire/release
+                //the same DataSource instance.
+                if (wrapper is IDataStreamPendingAccessWrapper pendingAccessWrapper
```

Comment:
You don't seem to use `pendingAccessWrapper` did you mean to?

Reply:
Good catch, turns out I didn't need it, removed.